packages/forms/src/model/form_array.ts

```diff
@@ -156,10 +156,10 @@ export class FormArray<TControl extends AbstractControl<any> = any> extends Abst
    * `valueChanges` observables emit events with the latest status and value when the control is
    * inserted. When false, no events are emitted.
    */
```

Comment:
Shouldn't this be updated respectively?

```suggestion
   * `onlySelf` observables emit events with the latest status and value when the control is
   * inserted. When false, no events are emitted.
   */
```

Reply:
You are right